packages/ui/src/lib/carousel/CarouselTest.svelte

```diff
@@ -1,15 +1,19 @@
 <script lang="ts">
 import Carousel from './Carousel.svelte';
+
+interface Item {
+  content: string;
+}
 </script>
 
 
-{#snippet card(cardItem: unknown)}
-<p>{cardItem}</p>
+{#snippet card(item: Item)}
+  <p>{item.content}</p>
 {/snippet}
 
 <div>
   <p class="text-lg first-letter:uppercase font-bold pb-5">Learning center:</p>
   <div class="bg-[var(--pd-content-card-bg)] p-4 rounded-lg">
-    <Carousel cards={['card 1', 'card 2', 'card 3']} {card} cardWidth={340} />
+    <Carousel cards={['card 1', 'card 2', 'card 3'].map((value) => ({ content: value }))} {card} cardWidth={340} />
```

Comment:
hello, not sure to get why we need to add a subfield `.content` rather than just keeping the `string[]` type

Reply:
I wanted to have a more complex example, but I can add both, or revert, I don't mind

Reply:
as it's not related to the changes of using generics, I would add a separate commit or remove that part

Reply:
np, reverted 👍